frameworks/management.rst

```diff
@@ -461,6 +466,53 @@ render these directly to JSON. Instead, it translates the responses to a more
 convenient JSON format reflecting specific types of requests. Several commands
 share a common output format.
 
+.. _zeek-client-installation:
+
+Standalone installation
+-----------------------
+
+As mentioned above, Zeek ships with ``zeek-client`` by default. Since users will
+often want to use the client from machines not otherwise running Zeek, the
+client is also available as a standalone Python package via ``pip``:
+
+.. code-block:: console
+
+   $ pip install zeek-client
+
+Users with custom Zeek builds who don't require a Zeek-bundled ``zeek-client``
+can skip its installation by configuring their build with
+``--disable-zeek-client``.
+
+.. _zeek-client-compatibility:
+
+Compatibility
+-------------
+
+Zeek 5.2 switched client/controller communication from Broker's native wire
+format to the newer `WebSocket data transport
+<https://docs.zeek.org/projects/broker/en/current/web-socket.html>`_, with
+``zeek-client`` 1.2.0 being the first version to exlusively use WebSockets.
+This has a few implications:
+
+* Sincer Broker dedicates separate ports to the respective wire formats, the
+  controller listens on TCP port 2149 for WebSocket-powered connections, while
+  TCP port 2150 remains available for connections by native-Broker clients, as well
+  as by management agents connecting to the controller.
+
+* ``zeek-client`` 1.2.0 and newer default to connecting to port 2149.
+
+* Controllers running Zeek older than 5.2 need tweaking to listen on a WebSocket
+  port, for example by saying:
+
+  .. code-block:: console
+
+     event zeek_init()
+         {
+         Broker::listen_websocket("0.0.0.0", 2149/tcp);
+         }
+
+* You can continue to use older clients with Zeek 5.2 and newer.
```

Comment:
The rest of the bullet points don't address the user with "you", so maybe the following, but no strong opinion.

* Older ``zeek-client`` versions continue to work with Zeek 5.2 and newer.

Reply:
Thanks! Done.